notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_llm_evaluation_for_summarization_task.ipynb

```diff
@@ -577,7 +579,7 @@
         "id": "veq26QZ7OMoC"
       },
       "source": [
-        "In this notebook, we will evaluate a third-party model's predictions (located in the `summary` column of `DATASET`) against the output of `text-bison@001` using a built-in summarization instruction. The task being performed is summarization.\n",
+        "In this notebook, we will evaluate a third-party model's predictions (located in the `summary` column of `DATASET`) against the output of `gemini-pro 1.0` using a built-in summarization instruction. The task being performed is summarization.\n",
```

Comment:
nit: `gemini-1.0-pro` or Gemini 1.0 Pro

Also, can we use 1.5 Flash or Pro instead? I think we're trying to encourage users to use 1.5

Reply:
Done. Tested with 1.5 pro on my side.